Tests/TokamakStaticHTMLTests/HTMLTests.swift

```diff
@@ -18,6 +18,149 @@
 import TokamakStaticHTML
 import XCTest
 
+private let expectedHTML =
```

Comment:
It would be nice to use https://github.com/pointfreeco/swift-snapshot-testing here instead, but we're running these tests in the Wasm environment, and file access is tricky there. Hardcoding the test fixture here for now.